src/xdocs/checks/annotation/annotationlocation.xml

```diff
@@ -240,16 +251,18 @@ class Example3 {
         <p id="Example4-code">Example:</p>
         <source>
 class Example4 {
-  @NotNull private boolean field1; //ok
-  @Override public int hashCode() { return 1; } //ok
-  @NotNull //ok
+  @NotNull private boolean field1; // ok as check is applied for METHOD_DEF
```

Comment:
`// ok as check is applied for METHOD_DEF` -> `// ok, since 'tokens' property set to METHOD_DEF only`, please always use language in check properties/documentation.